### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by mehtavishwa30**
The following proposed changes fix typos and make a few light changes in the design and grammar.

### CODE REVIEW

**Comment by judepereira** [comment]
Can you merge the Windows section and Python section together?

**Comment by mehtavishwa30** [comment]
Yes, that would make more sense.

**Comment by judepereira** [comment]
LGTM. @JeanBarriere to review and merge when ready.

File: quick-start/README.md [hunk 1]
```diff
@@ -15,7 +15,11 @@ export default {
 
 ## Our Story
 
-It started with questions any curious person should be asking: *Why is there so much complexity in development?* *How can we lower the bar of development?* *What is the future of programming?* High-level languages enabled millions of people to call themselves developers while simultaneously making development exponentially easier. We believe it's time to think beyond high-level languages and look towards a new language built on the cloud, for the cloud, that transforms the industry like high-level languages once did.
+It started with questions any curious person should be asking: 
+-*Why is there so much complexity in development?*
```
Comment (JeanBarriere) [inline quick-start/README.md]:
```suggestion
- *Why is there so much complexity in development?* 
```

File: quick-start/README.md [hunk 2]
```diff
@@ -15,7 +15,11 @@ export default {
 
 ## Our Story
 
-It started with questions any curious person should be asking: *Why is there so much complexity in development?* *How can we lower the bar of development?* *What is the future of programming?* High-level languages enabled millions of people to call themselves developers while simultaneously making development exponentially easier. We believe it's time to think beyond high-level languages and look towards a new language built on the cloud, for the cloud, that transforms the industry like high-level languages once did.
+It started with questions any curious person should be asking: 
+-*Why is there so much complexity in development?* 
+-*How can we lower the bar of development?*
```
Comment (JeanBarriere) [inline quick-start/README.md]:
```suggestion
- *How can we lower the bar of development?* 
```

File: quick-start/README.md [hunk 3]
```diff
@@ -15,7 +15,11 @@ export default {
 
 ## Our Story
 
-It started with questions any curious person should be asking: *Why is there so much complexity in development?* *How can we lower the bar of development?* *What is the future of programming?* High-level languages enabled millions of people to call themselves developers while simultaneously making development exponentially easier. We believe it's time to think beyond high-level languages and look towards a new language built on the cloud, for the cloud, that transforms the industry like high-level languages once did.
+It started with questions any curious person should be asking: 
+-*Why is there so much complexity in development?* 
+-*How can we lower the bar of development?* 
+-*What is the future of programming?*
```
Comment (JeanBarriere) [inline quick-start/README.md]:
```suggestion
- *What is the future of programming?* 
```

File: quick-start/README.md [hunk 4]
```diff
@@ -15,7 +15,11 @@ export default {
 
 ## Our Story
 
-It started with questions any curious person should be asking: *Why is there so much complexity in development?* *How can we lower the bar of development?* *What is the future of programming?* High-level languages enabled millions of people to call themselves developers while simultaneously making development exponentially easier. We believe it's time to think beyond high-level languages and look towards a new language built on the cloud, for the cloud, that transforms the industry like high-level languages once did.
+It started with questions any curious person should be asking: 
+-*Why is there so much complexity in development?* 
+-*How can we lower the bar of development?* 
+-*What is the future of programming?* 
+High-level languages enabled millions of people to call themselves developers while simultaneously making development exponentially easier. We believe it's time to think beyond high-level languages and look towards a new language built on the cloud, for the cloud, that transforms the industry like high-level languages once did.
```
Comment (JeanBarriere) [inline quick-start/README.md]:
```suggestion

High-level languages enabled millions of people to call themselves developers while simultaneously making development exponentially easier. We believe it's time to think beyond high-level languages and look towards a new language built on the cloud, for the cloud, that transforms the industry like high-level languages once did.
```

**Comment by mehtavishwa30** [comment]
Thanks for your review and suggestions, @JeanBarriere! Must've have missed that while previewing. Have committed the changes.

**Comment by JeanBarriere** [comment]
Then it's a good to go for me :)
Merging, and thanks again !

**Pull request merged**